airflow-core/docs/administration-and-deployment/plugins.rst

```diff
@@ -192,13 +194,28 @@ definitions in Airflow.
         "name": "Name of the Middleware",
     }
 
+    # Creating a iframe view that will be rendered in the Airflow UI.
+    iframe_view_with_metadata = {
+        "name": "Name of the Iframe View",
+        # Source URL of the iframe. This URL can be templated using context variables.
+        "src": "https://example.com/{DAG_ID}/{RUN_ID}/{TASK_ID}",
```

Comment:
I think `src` is more explicit than `url`.

Reply:
Do we have defined the elements that can be templated? Assuming it is in "Task Instance" it will be `DAG_ID`, `RUN_ID`, `TASK_ID`. `MAP_INDEX`? And if it is on "dag" level then only `DAG_ID`?